components/server/src/workspace/workspace-starter.ts

```diff
@@ -334,6 +334,7 @@ export class WorkspaceStarter {
                 // All they need access to is the base image repository, as we're building the Gitpod layer only.
                 const nauth = new BuildRegistryAuthSelective();
                 nauth.setAllowBaserep(true);
+                nauth.setAnyOfList(this.env.defaultBaseImageRegistryWhitelist);
```

Comment:
can you add some context? Like
                // Allow reopening workspaces built from the whitelisted registries.
                // Without the explicit authorization, workspaces only opens the first time after the build